### DISCUSSION TITLE
Logging in json

### DISCUSSION THREAD

**Discussion by vhppp** [bottlerocket-os/bottlerocket#2051] [Q&A]
Recently I've been trying Bottlerocket on aws-k8s-1.21. I've noticed that the individual pods are logging in plaintext rather than json in /var/log/pods and links are made to /var/log/containers. I suppose it depends on the implementation as to what is the norm. But standards are pushing for json. I do see some issues revolving around topic. But I am hopeful there is a workaround.

Is there anyway as of now for a user using EKS with bottlerocket to have containers log to files in json? I see the option for ecs but no option for those not using ecs.

**Comment by zmrow** [comment]
Hi there - Are you asking about individual pod logs or control plane logging?  I believe you're asking for control over individual pods logging?  If so, we don't have any control over their logging format. :(  

Let me know if you have additional details or more questions!

**Reply by vhppp** [reply]
Hi @zmrow, I'm asking actually about the logs from pods. They are coming as plain text lines. Would love for them to be json formatted lines
